repos/system_upgrade/el7toel8/actors/peseventsscanner/actor.py

```diff
@@ -56,3 +71,11 @@ def process(self):
         if to_install or to_remove:
             self.produce(RpmTransactionTasks(to_install=list(to_install), to_remove=list(to_remove)))
 
+        for pkg in to_install:
+            repo = pkg_repo.get(pkg, None)
+
+            if repo and repo not in REPOSITORIES_BLACKLIST:
```

Comment:
don't we want to remove pkgs from blacklisted repositories from to_install and inform user about them?

Reply:
Changed the code to handle that. By now, we are only using show_message with a very simple message to do not pollute user output. We will, probably, change that to use reporting capabilities in the future.